### DISCUSSION TITLE
Can't get  education benefits

### DISCUSSION THREAD

**Discussion by NhatErik76** [community/community#120939] [GitHub Education]
### Hi everyone,

Hey everybody.
After i have tried to process my application to get student benefits but after a week my email hasn't received any notifications.

i have 2 email. The first email is my personal email (mail email) and the second is school email (backup email)
please help me

**Comment by Manjunathhs-0003** [comment]
you can have two email in the same account
<img width="992" alt="Screenshot 2024-04-24 at 13 15 37" src="https://github.com/community/community/assets/146455138/630d38d3-5526-46d8-95fa-1df4e9b17ac5">
just like this, one acts as an educational email, and one visible email

**Comment by NhatErik76** [comment]
> even though i've followed the steps, my application behaves as if it's not subscribed. ![image](https://private-user-images.githubusercontent.com/160149794/325293839-016c0c50-d910-4c49-97a3-cd0e66e4ba70.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzQ4NTAsIm5iZiI6MTcxMzk3NDU1MCwicGF0aCI6Ii8xNjAxNDk3OTQvMzI1MjkzODM5LTAxNmMwYzUwLWQ5MTAtNGM0OS05N2EzLWNkMGU2NmU0YmE3MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjAyMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNDg4MzBiOTY3NWJjMGUxOGEwYjhhZDcyYTBiZmYxMTc4NTc4MTI2NWE3NzlhNzM0MmYzMmNiNDdjOTkwZjUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.amPEIfhZI5P-K2HRG2YGMOFykZn-1xurYtmu5f-zX1A) ![image](https://private-user-images.githubusercontent.com/160149794/325293947-d14d0632-40fc-4af7-a139-5cc297f5a827.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzQ4NTAsIm5iZiI6MTcxMzk3NDU1MCwicGF0aCI6Ii8xNjAxNDk3OTQvMzI1MjkzOTQ3LWQxNGQwNjMyLTQwZmMtNGFmNy1hMTM5LTVjYzI5N2Y1YTgyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjAyMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hYjc1ZDM5MjBhMDkwMjE0ZDQwOTEzNmM1NDEzMWVjMmI1NWFkODhmZDVkNWIyMTIyMTE0YzQ0NDcxZDk1YmUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.h5pNfITHmsmTUuZRcSovmRr06fszcjgDy-Q2SMSszgs) ![image](https://private-user-images.githubusercontent.com/160149794/325294060-2b1301d5-54f7-476a-a715-ddf25d1373c0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5NzQ4NTAsIm5iZiI6MTcxMzk3NDU1MCwicGF0aCI6Ii8xNjAxNDk3OTQvMzI1Mjk0MDYwLTJiMTMwMWQ1LTU0ZjctNDc2YS1hNzE1LWRkZjI1ZDEzNzNjMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNFQxNjAyMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mN2NlN2E2MGI0MzU3ZjgxYWE1Y2M1MjE3ODY4ZjU5YmQzZTg4YjA3YjVhNzVkYzVkMjI0MWFiNjVhODg5MWNlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.V86n0RGxJhMluFRm1ERMs34HMTBr8tIPdPU6BpQ0DL4) i'm frustrated becasue the group of github not replying

**Reply by Manjunathhs-0003** [reply]
just do one thing, promote your github profile somewhere inhere, the one who blocks it is a maintainer
so catch hold of them and ask queries to them 
simple

**Reply by Manjunathhs-0003** [reply]
because promoting is not allowed in github community platform

**Reply by samus-aran** [reply]
That is a very strange way to get a hold of a maintainer... I would recommend not doing that. 

You can usually tell who the Community Managers are: usually the authors of pinned posts.

**Reply by Manjunathhs-0003** [reply]
as if it is easy to find them

**Comment by soysantana** [comment]
## Possible Solution: GitHub Education Benefits

### Problem Understanding:
You have applied for student benefits but haven't received any email notifications regarding the status of your application on GitHub, despite it being a week since you applied. You have two email addresses: your personal email and your school email, and you're seeking help in resolving this issue specifically on GitHub.

### Proposed Solution:
To address the lack of email notifications for your GitHub application, follow these steps:

1. **Review GitHub Notification Settings:**
   - Log in to your GitHub account and navigate to the notification settings. Ensure that notifications are enabled for relevant events, such as pull request updates, comments, etc.

2. **Check Spam Folder:**
   - Verify the spam or junk folder in both your personal and school email accounts. GitHub notifications may have been incorrectly filtered.

3. **Verify Application Status on GitHub:**
   - Visit the GitHub page where you submitted your student benefits application. Check for updates or comments on the status of your application directly on the platform.

4. **Check Email Settings on GitHub:**
   - Review the email settings in your GitHub account to ensure that your email addresses are configured correctly and that there are no typographical errors.

5. **Update Notification Settings:**
   - If you find that notifications are disabled or configured incorrectly on GitHub, adjust them according to your preferences to ensure you receive notifications about the status of your application.

6. **Contact GitHub Support:**
   - If after reviewing all these steps you still do not receive notifications, contact GitHub's technical support to report the issue. Provide specific details about your application and associated email addresses so they can assist you better.

By following these steps, you should be able to address the issue of missing email notifications for your application on GitHub. Make sure to review notification settings, check spam folders, verify the application status on GitHub, and contact GitHub support if necessary for further assistance. This should help you receive the necessary notifications about the status of your student benefits application on GitHub.

---

### I hope it has been useful to you, if you have any questions write to me @soysantana

**Comment by samus-aran** [comment]
Hi @NhatErik76 

We can see you haven't completed your verification process. That is the reason you haven't received any notification email. Complete the application and your should see a status stating if it's being processed, Approved or completed (purple Approved).

**Reply by NhatErik76** [reply]
Hi @samus-aran 
Oh. I remember I have received email from github for identifying. If you suspect me, you can resend the email for me so that i can confirm again that i am a student.
